src/components/Checkbox/Checkbox.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { useEffect, useState } from 'react';
+import cn from 'classnames';
+
+import { ReactComponent as CheckmarkNormal } from './CheckmarkNormal.svg';
+import { ReactComponent as CheckmarkCompact } from './CheckmarkCompact.svg';
```

Comment:
Couldnt add comments on the files directly, so adding it here. 

These icons do not seem to be designed with the same specs as other icons we use. They should be designed on a 36*36 grid if I remember correctly.

An alternative to using SVG's for this is an approach similar to this: https://moderncss.dev/pure-css-custom-checkbox-style/ to avoid adding all the extra HTML elements - but that depends a bit on how easy it will be to replicate the look, and will probably need a bit discussion with UX

Reply:
I have recalculated the path point values to percentages and applied them to the clip-path property on the checkmark element. The visible result should be exactly the same as before.